andela/colossus-backend
# #167733825 Integrate The Request Table With The Backend

#### What does this PR do?
- This pull request creates the models and migrations of the requests and trips with their corresponding associations/references

#### Description of Task to be completed?
- Create the models and migrations using sequelize
- Configure the corresponding associations/references
- Implement an endpoint to query for all requests pertaining to a particular user
- Write a unit test for the endpoint

#### How should this be manually tested?
- Ensure you have git, node and  npm installed
- Clone the colossus-backend repo
- Open your terminal and navigate to the repo you've just cloned
- Enter the command 'npm install && npm test'

#### Any background context you want to provide?
- A user can make multiple requests and a request can have multiple trips tied to it

#### What are the relevant pivotal tracker stories?
- #167733825
